### PR TITLE
Add device compute capability info

### PR DESCRIPTION
**Pull request by anight**
Added CudaComputeCapability field to Device struct

### CODE REVIEW

File: bindings/go/nvml/bindings.go [hunk 1]
```diff
@@ -330,6 +330,16 @@ func (h handle) deviceGetDecoderUtilization() (*uint, error) {
 	return uintPtr(usage), errorString(r)
 }
 
+func (h handle) deviceGetCudaComputeCapability() (int, int, error) {
+	var major, minor C.int
+
+	r := C.nvmlDeviceGetCudaComputeCapability(h.dev, &major, &minor)
+	if r == C.NVML_ERROR_NOT_SUPPORTED {
```
[the remark this answers is not in the document]
Comment (anight) [reply bindings/go/nvml/bindings.go]: Sure thing

File: bindings/go/nvml/nvml.go [hunk 2]
```diff
@@ -175,18 +175,24 @@ type PCIInfo struct {
 	Bandwidth *uint
 }
 
+type CudaComputeCapabilityInfo struct {
+	Major int
+	Minor int
```
Comment (guptaNswati) [inline bindings/go/nvml/nvml.go]: `Major int*`
`Minor int*`

File: bindings/go/nvml/bindings.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (guptaNswati) [reply bindings/go/nvml/bindings.go]: Actually, change the return type of major and minor numbers to a pointer to int.
Update the function to:
`func (h handle) deviceGetCudaComputeCapability() (*int, *int, error) {`
	`var major, minor C.int`
	`r := C.nvmlDeviceGetCudaComputeCapability(h.dev, &major, &minor)`
         `if r != C.NVML_SUCCESS {`
                    `return nil, nil, errorString(r)`
        `}`
         `majorNum := int(major)`
         `minorNum := int(minor)`
       `return &majorNum, &minorNum, nil`
`}`

**Comment by guptaNswati** [comment]
Also before i merge your PR, you need to sign the CLA and sign-off yours commits.

Thanks

**Comment by guptaNswati** [comment]
Added this feature in a separate PR. Closing this.

**Pull request closed without merging**